styles.css

```diff
@@ -0,0 +1,114 @@
+html{
+    background-color: #f5eeee;
+    font-size: 16px;
+}
+
+
+
+h1{
+    color: #ff5566;
+    background-color:darkslategray;
+    text-align: center;
+    font-size: 3rem;
+    padding: 1rem;
+    font-family: "Impact", sans-serif;
+    margin: 0;
+    
+}
+
+nav{
+    background-color: darkslategray;
+    position: sticky;
+    top: 0rem;
+    
+}
+
+nav a{
+    position: relative;
+    color: white;
+    text-decoration: none;
+    font-size: 2rem;
+    font-family: "Verdana";
+    font-weight: bold;
+    padding: 1rem;
+    
+    
+    background-color: darkslategray;
+    border-radius: 0.6rem;
+    
+}
+
+nav a.current{
+    top: 0.8rem;
+}
+
+nav a:hover{
+    background-color: #223243;
+}
+
+main{
+    margin: 2rem auto;
+    max-width: 80rem;

```

Comment:
rem als max-width Angabe ist nicht sinnvoll. Lieber % oder vw/vh für viewport width and viewport height